plat/mediatek/common/mtk_sip_svc.c

```diff
@@ -67,6 +67,18 @@ static uint64_t mediatek_sip_handler(uint32_t smc_fid,
 		ret = mt_sip_pwr_mtcmos_support();
```

Comment:
You've set the execute permission on several files in this commit. Is that intended?